tests/c/pthread_create.c

```diff
@@ -1,27 +1,86 @@
 // Compiler:
 // Run-time:
-//   status: error
-//   stderr: ...
-//    not yet implemented: Allocate and set thread-local ShadowStack instance. No support for threads yet!
+//   stderr:
 //    ...
+//    ret1 = 0xdead
+//    ...
+//    oof2: {{_}} 20
+//    ...
+//    ret2 = 0xbaff1ed
+
+// This test checks that new threads create their own shadow stack. For this is
```

Comment:
nit: s/is/it/ on the end of the line

Reply:
[4115f6f](https://github.com/ykjit/yk/pull/1664/commits/4115f6f15d858560e7c7a756896f49d234672e13)